docker-compose.yml

```diff
@@ -34,3 +38,41 @@ services:
       # - 2135:2135
       - 2136:2136
     hostname: localhost
+  dashboard:
+    env_file: [.env_file]
+    build:
+      context: ./dff/utils/docker
+      dockerfile: dockerfile_stats
+    image: ghcr.io/deeppavlov/superset_df_dashboard:latest
+    ports:
+      - "8088:8088"
+  clickhouse:
+    env_file: [.env_file]
+    image: yandex/clickhouse-server:latest
+    restart: unless-stopped
+    ports:
+      - '8123:8123'
+      - '8443:8443'
+      - '9000:9000'
+    volumes:
+      - ch-data:/var/lib/clickhouse/
+  otelcol:
+    image: otel/opentelemetry-collector-contrib:latest
+    container_name: otel-col
+    restart: unless-stopped
+    command: [ "--config=/etc/otelcol-config.yml", "--config=/etc/otelcol-config-extras.yml" ]
+    volumes:
+      - ./dff/utils/otel/otelcol-config.yml:/etc/otelcol-config.yml
+      - ./dff/utils/otel/otelcol-config-extras.yml:/etc/otelcol-config-extras.yml
+    ports:
+      - "4317:4317"          # OTLP over gRPC receiver
+      - "4318:4318"     # OTLP over HTTP receiver
```

Comment:
Both comments should start at the same column.

Reply:
Done